ci/recipe/meta.yaml

```diff
@@ -20,6 +20,7 @@ requirements:
   run:
     - pandas
     - scikit-bio
+    - unifrac
```

Comment:
Looks like the recipe is missing `psutil`, too:

https://github.com/ChrisKeefe/q2-diversity-lib/runs/695274757?check_suite_focus=true#step:4:2274

Reply:
Oh weird - the diff that I was looking at didn't show `psutil`, but when I opened the PR up, there it is... My mistake!

Reply:
![Screen Shot 2020-05-21 at 8 27 55 AM](https://user-images.githubusercontent.com/274668/82575198-fe005880-9b3c-11ea-91a7-70751f9767f5.png)

Where did psutil go? Its a git mystery!!